lib/QubitInformationObjectXmlCache.class.php

```diff
@@ -81,12 +81,17 @@ public function export($resource)
    *
    * @return null
    */
-  public function exportAll()
+  public function exportAll($options = array())
   {
+    $skip = isset($options['skip']) ? $options['skip'] : 0;
```

Comment:
Do you have to do this? I'd assume `0` is the default if the user didn't pass the argument.